<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchTaggingOptionsWidget</class>
 <widget class="QWidget" name="BatchTaggingOptionsWidget">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="options_group">
     <property name="title">
      <string>⚙️ 일괄 태깅 옵션</string>
     </property>
     <layout class="QVBoxLayout" name="options_layout">
      <item>
       <widget class="QCheckBox" name="recursive_checkbox">
        <property name="text">
         <string>하위 디렉토리 포함</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ext_layout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>확장자:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ext_combo">
          <item>
           <property name="text">
            <string>모든 파일</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>이미지 파일</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>문서 파일</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>사용자 정의</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="custom_ext_edit">
          <property name="placeholderText">
           <string>.jpg,.png,.pdf</string>
          </property>
          <property name="maximumWidth">
           <number>150</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
